packages/strip/src/index.js

```diff
@@ -38,21 +38,34 @@ export default function strip(options = {}) {
   const sourceMap = options.sourceMap !== false;
 
   const removeDebuggerStatements = options.debugger !== false;
-  const functions = (options.functions || ['console.*', 'assert.*']).map((keypath) =>
```

Comment:
please avoid "personal preference" refactors like this. please merge your changes from lines 45-47 back to this code block and remove the extra `functionsPatterns` variable. this make look cleaner in your eyes, but it adds extra complexity to the code that's not necessary.

Reply:
ok